setup.py

```
@@ -831,6 +843,30 @@ def build_cmake(self, ext):
                                str(num_jobs)],
                               cwd=build_temp)
 
+    def build_pybind(self, ext, verbose: bool = True, pip_args: list[str] | None = None) -> None:
```

Ruff (0.13.1)

Unused method argument: `verbose`

(ARG002)